commands/namespace_get.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) OpenFaaS Author(s) 2023. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openfaas/faas-provider/types"
+	"github.com/spf13/cobra"
+)
+
+var namespaceGetCmd = &cobra.Command{
+	Use:   `get NAMESPACE_NAME`,
+	Short: "Get existing namespace",
+	Long:  "Get existing namespace",
+	Example: `faas-cli namespace get namespace-name
+	faas-cli namespace get namespace-name
+	faas-cli namespace get namespace-name --gateway=http://127.0.0.1:8080`,
+	RunE:    get_namespace,
+	PreRunE: preGetNamespace,
+}
+
+func init() {
+	namespaceCmd.AddCommand(namespaceGetCmd)
+}
+
+func preGetNamespace(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return fmt.Errorf("namespace name required")
+	}
+
+	if len(args) > 1 {
+		return fmt.Errorf("too many values for namespace name")
+	}
+
+	return nil
+}
+
+func get_namespace(cmd *cobra.Command, args []string) error {
+	client, err := GetDefaultSDKClient()
+	if err != nil {
+		return err
+	}
+
+	ns := args[0]
+
+	fmt.Printf("Getting Namespace: %s\n", ns)
+	res, err := client.GetNamespace(context.Background(), ns)
+	if err != nil {
+		return err
+	}
+
+	printNamespaceDetail(res)
+
+	return nil
+}
+
+func printNamespaceDetail(nsDetail types.FunctionNamespace) {
```

Comment:
Could you do this like faas-cli describe, or template/function store describe?

Reply:
Done